frontend/src/components/SamplesList.tsx

```diff
@@ -117,10 +56,22 @@ export const SamplesList: FunctionComponent<ISampleListProps> = ({
           : {
               first: max_rows,
             }),
-        options: {
+        sampleMetadataOptions: {
```

Comment:
Renaming to make this `options` argument more explicit as we're adding three more `options` arguments below.